correcting install for compilation details (#1623)
ROCmSoftwarePlatform/rocBLAS: docs/install/Linux_Install_Guide.rst

   sudo apt-get update
   sudo apt-get install rocblas

For Fedora, CentOS, or RHEL, use these commands:

.. code-block:: shell

   sudo dnf install rocblas

> If you are compiling a program that will link against rocBLAS, instead install the ``rocblas-dev`` package.
> Similarly at the top level instead of installing ``rocm``, install ``rocm-dev``.

Reviewer: ```suggestion
If you are compiling a program that will link against rocBLAS, instead install the ``rocblas-dev`` package.
Similarly, at the top level, install ``rocm-dev`` instead of ``rocm``.
```

For example, on Ubuntu or Debian, use these commands:

.. code-block:: shell

   sudo apt-get update
   sudo apt-get install rocblas-dev

For Fedora, CentOS, or RHEL, use these commands:

.. code-block:: shell

   sudo dnf install rocblas-devel

If you are using a different distribution, use the appropriate package manager to install the ``rocblas`` or ``rocblas-dev`` package.

The ``rocblas`` package contains the shared library and runtime only requirements.
The ``rocblas-dev`` package depends on the ``rocblas`` package that has the shared library and runtime requirements, but includes the header files and/or static library.

> The ``rocblas-dev`` package is not required to run the library, but it is required to compile code that uses rocBLAS.

Reviewer: ```suggestion
The ``rocblas-dev`` package is not required to run the library but is required to compile code that uses rocBLAS.
```

> After installation of the development package which includes headers, use rocBLAS like any other library with a C API.

Reviewer: ```suggestion
After installation of the development package which includes the headers, use rocBLAS like any other library with a C API.
```

CMake has a minimum version requirement, which is listed in the ``install.sh`` script.
See the ``--cmake_install`` flag in ``install.sh`` to upgrade automatically.

> The ROCm hipBLASLt dependency is not installed using ``-d`` flag to ``install.sh`` so install it

Reviewer: ```suggestion
The ROCm hipBLASLt dependency is not installed when using ``-d`` flag with ``install.sh``, so install it
```
